Formula/kubeval.rb

```diff
@@ -0,0 +1,33 @@
+class Kubeval < Formula
+  desc "Validate Kubernetes configuration files, supports multiple Kubernetes versions"
+  homepage "https://www.kubeval.com/"
+  url "https://github.com/instrumenta/kubeval.git",
+      tag:      "v0.16.1",
+      revision: "f5dba6b486fa18b9179b91e15eb6f2b0f7a5a69e"
+  license "Apache-2.0"
+  head "https://github.com/instrumenta/kubeval.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+      -X main.commit=#{Utils.git_head}
+      -X main.date=#{time.iso8601}
+    ].join(" ")
+    system "go", "build", *std_go_args(ldflags: ldflags)
+
+    pkgshare.install "fixtures"
+  end
+
+  test do
+    system bin/"kubeval", pkgshare/"fixtures/valid.yaml"
+    assert_equal 0, $CHILD_STATUS.exitstatus
```

Comment:
Is there any output here?

Reply:
I dont see any output, that is why I assert the `valid` config check status.

Reply:
I believe `system` call will fail the test if the command returns non-zero status, so this line might not even be needed.

Reply:
That sounds about right, just removed the return status check.